packages/api-openfin/src/window.js

```diff
@@ -106,23 +105,18 @@ class Window {
 
     MessageService.subscribe('*', 'test2', (message) => console.log(message));
 
-    const convertedOptions = convertOptions(options);
-    const mergedOptions = Object.assign(
-      {},
-      convertedOptions,
-      options
-    );
+    const openFinOptions = convertOptions(options);
```

Comment:
[this constant](https://github.com/ScottLogic/containerjs/pull/67/files#diff-e05e841065008af4a1271781d72f7dbbR65) and this constant should have consistent naming (i.e. `openfin` vs `openFin`)